src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -177,7 +177,7 @@
     <value>Vertical spacing</value>
   </data>
   <data name="MeasureTool_UnitsOfMeasure.Header" xml:space="preserve">
-    <value>Units of measurement</value>
+    <value>Extra units of measurement</value>
   </data>
   <data name="MeasureTool_UnitsOfMeasure_Pixels.Content" xml:space="preserve">
     <value>Pixels</value>
```

Comment:
If pixels are always shown, I imagine this should be "Show only pixels" instead?